ga4gh/backend.py

```diff
@@ -39,10 +39,75 @@ def parsePageToken(self, pageToken, numValues):
         # TODO define exceptions.InvalidPageToken and raise here.
         if len(tokens) != numValues:
             raise Exception("Invalid number of values in page token")
-        # TODO catch a ValueError here when bad integers are passed and
-        # convert this into the appropriate InvalidPageToken exception.
-        values = map(int, tokens)
-        return values
+        return tokens
+
+    def search(self, request, depth):
```

Comment:
This is really two different functions with depth being used to choose between them.
Separate them instead.